telegram/utils/helpers.py

```diff
@@ -139,3 +139,43 @@ def effective_message_type(entity):
             return i
 
     return None
+
+
+def create_deep_linked_url(bot_username, payload=None):
+    """
+    Creates a deep-linked URL for this ``bot_username`` with the specified ``payload``.
+    See  https://core.telegram.org/bots#deep-linking  to learn more.
+
+    The ``payload`` may consist of the following characters: ``A-Z, a-z, 0-9, _, -``
+
+    Note: Works well in conjunction with
+    ``CommandHandler("start", callback, Filters.regex('payload')``.
```

Comment:
Missing a paramthesis here?
Also shouldn't the filter be passed as a keyword argument?